assets/settings/components/SettingsApp.jsx

```diff
@@ -86,6 +91,32 @@ class SettingsApp extends React.Component {
                             errors={this.props.errors}
                             dispatch={this.props.dispatch}
 
+                        />
+                    </div> : null }
+                {this.props.selectedMenu === 'products' ?
+                    <div className="content">
+                        <ProductBar
+                            onNewProduct={this.props.newProduct}
+                            fetchItems={this.props.fetchItems}
+                        />
+                        <Products
```

Comment:
imo settings app should handle the routing/tabs, but should not be aware about all those Products properties. pls use connect with Products and handle it there